.gitignore

```diff
@@ -1,2 +1 @@
-dist/
```

Comment:
Did we want to ignore this *including* the files added here already? In that case you will probably need to do a `git rm dist`

Reply:
No. We didn't have `dist` before in git and I want to include it because that's the file that'll be used by the action rather than installing and requiring the dependencies at runtime.